runtimes/huggingface/mlserver_huggingface/common.py

```diff
@@ -38,12 +38,17 @@ class Config:
         env_prefix = ENV_PREFIX_HUGGINGFACE_SETTINGS
 
     task: str = ""
+    task_suffix: str = ""
     pretrained_model: Optional[str] = None
     pretrained_tokenizer: Optional[str] = None
     optimum_model: bool = False
     device: int = -1
     batch_size: Optional[int] = None
 
+    @property
+    def task_name(self):
```

Comment:
Could we also add a test for the `task_name` property? This is mainly to ensure no one changes it randomly in the future.